kratos/includes/model_part.h

```diff
@@ -732,23 +730,13 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     MasterSlaveConstraint::Pointer CreateNewMasterSlaveConstraint(const std::string& ConstraintName,
                                                                                     IndexType Id,
                                                                                     NodeType& rMasterNode,
-                                                                                    const DoubleVariableType& rMasterVariable,
+                                                                                    const Variable<double>& rMasterVariable,
```

Comment:
This is fine .. but just curious why you removed a `typedef` and decided to use` Variable<double>` ?

Reply:
:tipping_hand_man: 

I think it is cleaner this way, modelpart is already an overloaded class

Reply:
I can also do it the other way round, don't have a strong opinion

Reply:
reverted it, seems to be used in other places too :D